canonicalwebteam/discourse/parsers.py

```diff
@@ -733,41 +780,132 @@ def _calculate_remaining_duration(self, total_duration, sections):
 
         return sections
 
-    def _get_section(self, soup, title_text):
+
+class EngageParser:
```

Comment:
Can we split this into own files. canonicalwebteam/discourse/parsers/engageParser.py`
almost 1000 lines on a file starts to be extra long and difficult to work on.

Reply:
Done